src/__tests__/integration/settings.test.js

```diff
@@ -0,0 +1,54 @@
+import { integrationTest, waitFor, uploadFile, getNextStepButton, getTestFile } from './index';
+import { act } from 'react-dom/test-utils';
+import React from 'react';
+import Settings from '../../components/layouts/pages/settings/Settings';
+
+const TEST_FILE = getTestFile();
```

Comment:
Why simply not move the whole constant to `index`?

Reply:
Fixed